utils/build/docker/python_http/parametric/apm_test_client/server.py

```diff
@@ -245,10 +245,14 @@ def otel_start_span(args: OtelStartSpanArgs):
     else:
         parent_span = None
 
+    # Python SpanKind enum starts at 0 whereas the proto spec used here starts at 1
```

Comment:
This file no longer exists. The python_http app was removed. You should update your branch